### ISSUE TITLE
Automatically adding tags from the report

### ISSUE THREAD

**Issue by vinayhpandya** [Pavankunchala/LLM-Learn-PK#1]
Is it possible to automatically add tags in the markdown file? -> (Like what ML techniques, data collection techniques) are used in the paper. Also is it possible to add more custom prompts such as "What retrieval techniques does this paper use?" in a search field.

**Comment by Pavankunchala** [comment]
yeah you can, you can give few shot examples in  the main  prompt on how the output want to look like probably 3 to 5 would be more then enough , you can ask chatgpt or any other ones to generate you a few and add them in your main prompt
